src/services/provider/administration.provider.js

```diff
@@ -22,19 +22,15 @@ export const loadBvvAdministration = async (coordinate3857) => {
 
 	const result = await httpService.get(`${url}/${coordinate3857[0]}/${coordinate3857[1]}`);
 
-	if (result.ok) {
-		const payload = await result.json();
-		const community = payload.gemeinde;
-		const district = payload.gemarkung;
-		// later we have to add this for BAplus users
-		// const parcelDenomination = payload.flstBezeichnung;
-		if (community && district) {
-			const bvvAdministration = {
-				community: community,
-				district: district
+	switch (result.status) {
+		case 200: {
+			const { gemarkung: community, gemarkung: district } = await result.json();
```

Comment:
CopyPasta!

should be: `const { gemeinde: community, gemarkung: district } = await result.json();`